app/services/idv/session.rb

```diff
@@ -140,7 +142,15 @@ def build_profile_maker(user_password)
         applicant: applicant,
         user: current_user,
         user_password: user_password,
+        reproof_at: reproof_at,
       )
     end
+
+    # @return [Date,nil]
+    def reproof_at
+      date_str = user_session.dig('idv/doc_auth', :reproof_at) # via flow_session[:reproof_at]
```

Comment:
High level flow step 5a: reading `reproof_at` from the `flow_session` we can save it to the `Profile` (from step 2)